### PR TITLE
chore: change company for jmeridth (JM)

### PR DESCRIPTION
**Pull request by jmeridth**
Change from Procore to GitHub

I switched jobs in 3/2024

Also, I go by JM, changed name to match.

### CODE REVIEW

**Review by agilgur5 — APPROVED** [review]
You probably want to change your CNCF affiliation too if you haven't already. For unscoped(?) maintainers that is done here: https://github.com/cncf/foundation/blob/88f1a47/project-maintainers.csv. I believe when unspecified it attempts to use the `git` author email (which _can_ be flawed if not verified by PGP etc; I think someone accidentally used mine in a fork at some point in time for instance and CNCF's automation picked up on that)

**Comment by agilgur5** [comment]
ah right, DCO check is [failing](https://github.com/dcoapp/app/issues/211) in this repo too and I don't have override permissions.

**Review by todaywasawesome — APPROVED** [review]

**Review by terrytangyuan — APPROVED** [review]

**Comment by terrytangyuan** [comment]
Congrats on the new job and I hope you'll still stay involved!

File: MAINTAINERS.md [hunk 1]
```diff
@@ -40,7 +40,7 @@
 | Justin Marquis            | [34fathombelow](https://github.com/34fathombelow)       | Approver(docs/ci) - CD                                 | [Akuity](https://akuity.io/)                         |
 | Alexander Matyushentsev   | [alexmt](https://github.com/alexmt)                     | Lead - CD, Rollouts <br/>Approver - Workflows          | [Akuity](https://akuity.io/)                         |
 | Marco Maurer              | [mkilchhofer](https://github.com/mkilchhofer)           | Approver(helm-chart) - CD, Events, Rollouts, Workflows | [Swiss Post](https://post.ch/)                       |
-| Jason Meridth             | [jmeridth](https://github.com/jmeridth)                 | Approver(helm-chart) - CD, Events, Rollouts, Workflows | [Procore](https://www.procore.com/)                  |
+| JM (Jason Meridth)        | [jmeridth](https://github.com/jmeridth)                 | Approver(helm-chart) - CD, Events, Rollouts, Workflows | [GitHub](https://github.com/)                  |
```
Comment (crenshaw-dev) [inline MAINTAINERS.md:43]:
```suggestion
| JM (Jason Meridth)        | [jmeridth](https://github.com/jmeridth)                 | Approver(helm-chart) - CD, Events, Rollouts, Workflows | [GitHub](https://github.com/)                        |
```

**Pull request merged**